### PR TITLE
chore(release): 8.53.0

### PR DESCRIPTION
**Pull request by balzss**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,28 @@
...
 
+# [8.53.0](https://github.com/instructure/instructure-ui/compare/v8.52.0...v8.53.0) (2024-02-08)
+
+
+### Bug Fixes
+
+* **ui-editable:** fix the Esc key event propagation inside the Editable component ([5223f2f](https://github.com/instructure/instructure-ui/commit/5223f2ff3c4c3d3cc8c9bea32efbdcac3aebdfa3))
+* **ui-overlays:** add maskcounter ([fbe4d9e](https://github.com/instructure/instructure-ui/commit/fbe4d9eeaff0699068e43f5b3bb0afb05a206070))
+* **ui-text-area:** correct highligh on textarea when using themeOverride ([a80dd9a](https://github.com/instructure/instructure-ui/commit/a80dd9a81758ed6931ef0b503b86e9407b0090ca))
+* **ui-text-area:** correct highligh on textarea when using themeOverride ([b657193](https://github.com/instructure/instructure-ui/commit/b6571933c580835e23f0875a2cf385813962db00))
```
Comment (HerrTopi) [inline CHANGELOG.md:14]: these should be consolidated into one announcement imho

**Review by HerrTopi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (balzss) [reply CHANGELOG.md:14]: yes, nice catch

**Review by HerrTopi — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (joyenjoyer) [reply CHANGELOG.md]: Minor typo highligh-> highlight.

**Review by joyenjoyer — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (balzss) [reply CHANGELOG.md]: I will fix it in the release and announcement

**Pull request merged**